openshift/pipelines/Jenkinsfile.cicd

```diff
@@ -358,6 +358,64 @@ pipeline {
         // """
       }
     }
+    stage("Tag API") {
+      environment {
+        COMP_NAME = "api"
+      }
+      steps {
+        script {
+          openshift.withCluster() {
+            openshift.withProject() {
+              try {
+                timeout(time: 1, unit: "MINUTES") {
```

Comment:
I think this now expects us to enter a tag for all pipelines that use this Jenkinsfile.  Do we want to do this in our other environments?  I'd think for DEV we'd just want to keep using the `dev` tag.

Reply:
Right, let me explain exactly what this is doing.
1) If you don't enter a tag the current functionality of the pipeline is unchanged.
2) If you enter a tag, a new tag will be created in addition to any tags normally created by the pipeline. The new phases is ignored by all future phases of the pipeline, but it is essentially "saved" for historic reference.